agentlightning/instrumentation/agentops.py

```diff
@@ -3,28 +3,59 @@
 from __future__ import annotations
 
 import logging
-import multiprocessing
-import signal
-import socket
-import time
-from typing import Any, Callable
+from typing import Any, Callable, cast
 
-import flask
 import requests
-import setproctitle
+from agentops.client.api import V3Client, V4Client
+from agentops.client.api.types import AuthTokenResponse
+from agentops.sdk.exporters import AuthenticatedOTLPExporter
+from opentelemetry.exporter.otlp.proto.http.metric_exporter import OTLPMetricExporter
+from opentelemetry.exporter.otlp.proto.http.trace_exporter import OTLPSpanExporter
+from opentelemetry.sdk.metrics.export import MetricExportResult
+from opentelemetry.sdk.trace.export import SpanExportResult
 
 logger = logging.getLogger(__name__)
 
 __all__ = [
     "instrument_agentops",
     "uninstrument_agentops",
-    "agentops_local_server",
-    "AgentOpsServerManager",
 ]
 
 # Module-level storage for originals
 _original_handle_chat_attributes: Callable[..., Any] | None = None
 _original_handle_response: Callable[..., Any] | None = None
+_switch = False
+
+
+def set_switch(value: bool):
```

Comment:
rename it as something like bypass_agentops_service(enabled: bool)

Now I look at it, switch is not a good name. Very confusing.